plugins/cost-insights/src/components/BarChart/BarChartLegend.tsx

```diff
@@ -21,11 +21,12 @@ import { currencyFormatter } from '../../utils/formatters';
 import { CostInsightsTheme } from '../../types';
 import { useBarChartLayoutStyles as useStyles } from '../../utils/styles';
 
-type BarChartLegendOptions = {
+export type BarChartLegendOptions = {
```

Comment:
@ryanvazquez QQ for you, I noticed we have both `BarChartLegendOptions` & `BarChartOptions` which are the same thing, but `BarChartLegendOptions` is only used w/in the legend while the `options` passed into the legend are typed as `BarChartOptions` even though it expects `BarChartLegendOptions` 🤔Should we be switching those to `BarChartLegendOptions` or is there a reason to keep it that way ?

Reply:
Nope, those should share the same type. 😅